.github/workflows/graalpy-micronaut-guide.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '23.0.2'
+          java-version: '23.0.0'
```

Comment:
Why downgrade to `23.0.0`?

Reply:
Because otherwise we get only fallback runtime, Micronaut uses 24.0.0, not 24.0.2 and Truffle will not allow optimized runtime with that mismatch. IIRC in 25.0 this will be relaxed and the patch level will be ignored in the version check, then we can update